src/components/Page/components/Header/Header.tsx

```diff
@@ -136,9 +137,24 @@ export function Header({
         {
           primary,
         },
-      )}
-    </ConditionalWrapper>
-  ) : null;
+      );
+    } else {
+      content = primaryAction;
+    }
+
+    return (
+      <ConditionalWrapper
```

Comment:
If you're writing if statements above then doing an early return of `if (newDesignLanguage) { return content; `}` might be more readable than using `ConditionalWrapper`